src/ui/profile/ProfileSelection.js

```diff
@@ -0,0 +1,170 @@
+// eslint-disable-next-line no-redeclare
+/* global chrome */
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/profile.module.css';
+import AddProfile from './AddProfile';
+import BeyondLoader from './BeyondLoader';
+import Profiles from './Profile';
+
+const LoadedProfile = (profileModel) => ({
```

Comment:
By convention, you should only capitalize names for components and classes. When I see these functions being called I immediately think it's a class being called without `new`, but it's just a regular function.

Having said that, if you like doing it this way, it's not a huge deal.